Resources/Prototypes/Datasets/Names/military.yml

```
@@ -19,45 +19,45 @@
-  - Rigel
-  - Sargas
-  - Schedar
-  - Shaula
-  - Sirius
-  - Tabit
-  - Taurus
-  - Ukdah
-  - Vega
-  - Yildun
-  - Zosma
+  - Ажена
```

yamllint (1.29.0-1)

[error] wrong indentation: expected at least 3

(indentation)